common/djangoapps/student/views.py

```diff
@@ -145,7 +145,7 @@
 }
 REGISTRATION_UTM_CREATED_AT = 'registration_utm_created_at'
 # used to announce a registration
-REGISTER_USER = Signal(providing_args=["user", "profile"])
+REGISTER_USER = Signal(providing_args=["user", "registration"])
```

Comment:
Are you sure the only listener for `REGISTER_USER` is sailthru? Please double check